### PR TITLE
77: FHIR Patient Demographics

### PR DESCRIPTION
**Pull request by halprin**
# FHIR Patient Demographics

- Changed all order stuff to demographics.
  - The name of the controller and its methods and variables.
  - The endpoint name and the OpenAPI spec.
  - The e2e tests.
- Changed the HTTP response to return the FHIR resource ID and patient ID.
- Added the HAPI FHIR library
  - Using it with FHIR path to get certain elements of a Patient resource. 

## Issue

#77.

### CODE REVIEW

**Review by tjohnson7021 — APPROVED** [review]

**Comment by halprin** [comment]
Thanks for the review, @tjohnson7021.  @jorg3lopez, please review this too since Tiff and I both worked on it a bit.

File: e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              "telecom": [
+                {
+                  "system": "phone",
+                  "value": "+31201234567",
+                  "use": "mobile"
+                }
+              ]
+            }
+          ]
+        }"""
```
Comment (jorg3lopez) [inline e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy:95]: Wonderful work. FHIR is very robust, what are your thoughts on using files where we put the mockMessage for the e2e tests to use?

**Review by jorg3lopez — APPROVED** [review]
Great job!

File: e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy (hunk 1 above)
Comment (halprin) [reply e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy:95]: I like this idea.  Let's do that in some follow-on PR.

**Pull request merged**